src/Haskell/Ide/Engine/Plugin/Package.hs

```diff
@@ -87,15 +85,8 @@ type Package = T.Text
 -- May fail if no project dependency specification can be found.
 -- Supported are `*.cabal` and `package.yaml` specifications.
 -- Moreover, may fail with an IOException in case of a filesystem problem.
-addCmd :: CommandFunc AddParams J.WorkspaceEdit
-addCmd = CmdSync addCmd'
-
--- | Add a package to the project's dependencies.
--- May fail if no project dependency specification can be found.
--- Supported are `*.cabal` and `package.yaml` specifications.
--- Moreover, may fail with an IOException in case of a filesystem problem.
```

Comment:
Should this be removed?

Reply:
This is one of the benefits of removing the `CommandFunc` wrapper, as it allows us to coalesce all the `xCmd` `xCmd'` functions into just one. This should be an NFC